### DISCUSSION TITLE
Blade: how to retrieve Taxonomy value

### DISCUSSION THREAD

**Discussion by spaceageliving** [statamic/cms#6273] [General]
I have a taxonomy field in my entry blueprint:

![taxonomy-1](https://user-images.githubusercontent.com/874600/176753065-a7de295d-b507-4ebc-8c80-2a78e965258d.jpg)

The taxonomy looks like this:

<img width="995" alt="Screen Shot 2022-06-30 at 12 30 41 PM" src="https://user-images.githubusercontent.com/874600/176753172-1bd3a442-590f-422c-aba6-c3b40414a0c3.png">

The entry looks like this:

![Screen-Shot-2022-06-30-at-12 29 44-PM](https://user-images.githubusercontent.com/874600/176753227-b570a851-d2db-4e6d-9b94-0d6219c9421a.jpg)

How do I retrieve the taxonomy value for this entry in blade?

I have the value injected in the template at $page->speed_rpm, but can't seem to access any values on the collection. I'm thinking I need to do an inline Statamic::tags query using the slug? If so, how to access the slug value?

**Comment by jasonvarga** [comment]
Just like your previous discussion, make sure that field has `max_items: 1`.

**Reply by spaceageliving** [reply]
Success! Thanks @jasonvarga, much appreciated.

For some reason, the control panel isn't saving the max_items field to my yaml, but adding manually works fine.

**Comment by spaceageliving** [comment]
@jasonvarga OK I'm having this issue again...I thought I had a better handle after our last discussion, but this just isn't resolving for me:

I changed a taxonomy field from no max items to max_items: 1. The Yaml looks like this:
<img width="335" alt="Screen Shot 2022-08-13 at 7 46 15 PM" src="https://user-images.githubusercontent.com/874600/184519120-790982ab-16c4-4ca3-945e-38b3849d2a11.png">

An entry for the collection:
<img width="340" alt="Screen Shot 2022-08-13 at 7 46 58 PM" src="https://user-images.githubusercontent.com/874600/184519139-206c20ea-347b-4cc8-825c-d3bfbc9b3ab2.png">

My blade template:
<img width="264" alt="Screen Shot 2022-08-13 at 7 47 27 PM" src="https://user-images.githubusercontent.com/874600/184519162-380885d9-ec23-4823-ac3c-132203d75286.png">

The result:
<img width="905" alt="Screen Shot 2022-08-13 at 7 48 03 PM" src="https://user-images.githubusercontent.com/874600/184519189-50250657-7715-4844-9b4f-7a4ab4a16cd9.png">
<img width="1345" alt="Screen Shot 2022-08-13 at 7 48 35 PM" src="https://user-images.githubusercontent.com/874600/184519202-1a28a301-0da5-422e-add1-08a0a271762e.png">

Its as if after I made the change to max_items:1, its not taking effect.

I cleared the cache, stache....am I missing some additional step needed to "trigger" this change?

**Reply by jasonvarga** [reply]
`$page->music_genre` would be the term object itself. You should just ask for the appropriate field like the title or slug.

```blade
{{ $page->music_genre->title }}
```

Edit: I see you already worked that out in another comment. 👍

**Reply by spaceageliving** [reply]
Yes indeed, thank you @jasonvarga.

**Comment by spaceageliving** [comment]
dumping the $page:

<img width="478" alt="image" src="https://user-images.githubusercontent.com/874600/184519694-7e885968-cf3c-483e-b80c-6763cbe74737.png">

**Comment by spaceageliving** [comment]
OK I think I got this one resolved. Since its a taxonomy term, has to be resolved, so changed to:

{{ $page->music_genre->title }}

all good.
